External/docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# This is the Dockerfile for the Enthought Edge "external" example.
+# 
+# We use "edm-rockylinux-8" as the base image.  This is a Rockylinux-8 based
+# image which includes EDM.
+#
+# EDM dependencies for the app are brought in via a .zbundle file.  This avoids
+# the need to pass your EDM token and/or a custom edm.yaml into the Dockerfile.
+#
+# We perform a two-stage build, to avoid including the .zbundle in the layers
+# of the published image.
+
+# First stage
+
+ARG BASE_IMAGE=quay.io/enthought/edm-rockylinux-8:latest
+
+FROM $BASE_IMAGE AS stage_one
+
+ARG EDGE_BUNDLE=app_environment.zbundle
+COPY $EDGE_BUNDLE /tmp/app_environment.zbundle
```

Comment:
the repo misses the instructions to create the zbundle (or are they outside of the PR?)

Reply:
I'm wrong. They are in the devspace.yaml file (https://github.com/enthought/edge-examples/pull/90/files#diff-cb14dc46e27f90d840df8f70ad0590ba96d372ab65af1389ff285a41e8b3b841)